File: MiYou.API/Models/Admin/Client/ClientResponse.cs
using MiYou.Shared.Enums;

namespace MiYou.API.Models.Admin.Client
{
    public class ClientResponse
    {
        public int Id { get; set; }
        public DateOnly ClientSince { get; set; }
        public string ContactPerson { get; set; }
        public bool Active { get; set; }
        public string FullName { get; set; }
        public PaymentStatusEnum PaymentStatus { get; set; }
    }

    public class ClientListResponse
    {
        public List<ClientResponse> ClientList { get; set; } = new List<ClientResponse>();
    }
}

File: MiYou.Web/src/app/models/admin/client/client.response.ts
import { PaymentStatus } from "../../../utilities/enums/paymentStatus.enum";

export interface ClientResponse{
    id: number,
    fullName: string,
    clientSince: Date,
    contactPerson: string,
    active: boolean,
    paymentStatus: PaymentStatus
}

export interface ClientListResponse{
    clientList: ClientResponse[];
}